test/lit/wasm-split/invalid-options.wast

```diff
@@ -0,0 +1,64 @@
+;; Test that invalid command line option combinations are properly rejected with
+;; helpful error messages.
+
+;; --instrument cannot be used with --profile
+;; RUN: not wasm-split %s --instrument --profile %t 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-PROFILE
+
+;; --instrument cannot be used with -o1
+;; RUN: not wasm-split %s --instrument -o1 %t 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-OUT1
+
+;; --instrument cannot be used with -o2
+;; RUN: not wasm-split %s --instrument -o2 %t 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-OUT2
+
+;; --instrument cannot be used with --import-namespace
+;; RUN: not wasm-split %s --instrument --import-namespace=foo 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-IMPORT-NS
+
+;; --instrument cannot be used with --placeholder-namespace
+;; RUN: not wasm-split %s --instrument --placeholder-namespace=foo 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-PLACEHOLDER-NS
+
+;; --instrument cannot be used with --export-prefix
+;; RUN: not wasm-split %s --instrument --export-prefix=foo 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-EXPORT-PREFIX
+
+;; --instrument cannot be used with --keep-funcs
+;; RUN: not wasm-split %s --instrument --keep-funcs=foo 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-KEEP-FUNCS
+
+;; --instrument cannot be used with --split-funcs
+;; RUN: not wasm-split %s --instrument --split-funcs=foo 2>&1 \
+;; RUN:   | filecheck %s --check-prefix INSTRUMENT-SPLIT-FUNCS
+
+;; --instrument is required to use -o
+;; RUN: not wasm-split %s -o %t 2>&1 \
+;; RUN:   | filecheck %s --check-prefix NO-INSTRUMENT-OUT
+
+;; --instrument is required to use --profile-export
+;; RUN: not wasm-split %s --profile-export=foo 2>&1 \
+;; RUN:   | filecheck %s --check-prefix NO-INSTRUMENT-PROFILE-EXPORT
+
+;; INSTRUMENT-PROFILE: error: --profile cannot be used with --instrument
+
+;; INSTRUMENT-OUT1: error: primary output cannot be used with --instrument
```

Comment:
I thought the big benefit of lit is that the output is next to the case. why are all the outputs at the end?

Reply:
The more general benefit of lit is its flexibility and ability to put multiple test invocations in a single file ;)

In this case it seemed simpler to put the error messages all together because they are so simple to read. Do you think this would be more readable with the error messages next to their corresponding RUN lines?

Reply:
I think that might be clearer. but it might be that i'm just not used to lit in general since I don't do much work on LLVM. it's probably fine as is and I'll get used to it.